pkg/services/ngalert/api/test-data/ruler-loki-recipient.http

```diff
@@ -0,0 +1,75 @@
+@lokiDatasourceID = 32
+
+// should point to an existing folder named alerting
+@namespace1 = test
+
+// create/update test namespace group42 rulegroup
+POST http://admin:admin@localhost:3000/ruler/{{lokiDatasourceID}}/api/v1/rules/{{namespace1}}
```

Comment:
These ones don't have the api prefix?

Reply:
They should have now that #32716 is merged to this one